ocw/lib/vault.py

```diff
@@ -41,11 +41,14 @@ def revoke(self):
                 raise e
         finally:
             self.auth_json = None
+            self.client_token = None
+            self.client_token_expire = None
 
     def getClientToken(self):
         if self.isClientTokenExpired():
-            r = self.httpPost('/v1/auth/userpass/login/' + self.user, data={'password': self.password})
-            j = r.json()
+            self.client_token = None
+            self.client_token_expire = None
+            j = self.httpPost('/v1/auth/userpass/login/' + self.user, data={'password': self.password}).json()
```

Comment:
currently `httpPost` is called 3 times in our code , one time we not collecting return value at all and in 2 other cases we require JSON after all . What if we will just return JSON from `httpPost`  ?

Reply:
I think I will keep it for now. As I find `.json()` more readable. If changing it, I would like to rename the function as well, maybe like `httpJsonGet()` ?